dist/tools/pyterm/pyterm

```diff
@@ -138,7 +138,7 @@ class SerCmd(cmd.Cmd):
 
         # create Logging object
         my_millis = "%.4f" % (time.time())
-        date_str = '%s.%s' % (time.strftime('%Y%m%d-%H:%M:%S'), my_millis[-4:])
+        date_str = '%s.%s' % (time.strftime('%Y%m%d-%H.%M.%S'), my_millis[-4:])
```

Comment:
Yes, it's somewhat unrelated, but the variable was not evaluated before anyway. But AFAIK MS Windows cannot handle colon in file names.